src/lib/schemas/translateValueType.ts

```diff
@@ -0,0 +1,36 @@
+import i18n from '@dhis2/d2-i18n'
+
+const translations: { [key: string]: string } = {
+    MULTI_TEXT: i18n.t('MULTI_TEXT'),
+    TEXT: i18n.t('Text'),
+    LONG_TEXT: i18n.t('Long text'),
+    LETTER: i18n.t('Letter'),
```

Comment:
Already have these https://github.com/dhis2/maintenance-app-beta/blob/master/src/constants/translatedModelConstants.ts

Reply:
I removed my duplicate translations. I can't use your helpers until you merged, so will adjust code once your changes are in master